README.md

```diff
@@ -70,37 +70,48 @@ dependencies {
 
 > Checkout the custom thumbnail layout sample to see more detail [here](https://github.com/nimblehq/recent-apps-thumbnail-hiding/blob/master/app/src/main/res/layout/activity_main.xml#L26-L33)
 
-### Low API support (25 and lowers)
+### Exceptions handling
 
-The core approach `HardwareKeyWatcher` in this
-lib [doesn't work on API 25 and lowers](https://docs.google.com/spreadsheets/d/1znmSllEYHuOhmla7EWFXYeWuv1EZQiVkB9Mibhcj52s/edit?usp=sharing)
-. In order to provide an option to cover the hiding app thumbnail on more and more devices, this lib adds support to
-apply [FLAG_SECURE](https://developer.android.com/reference/android/view/WindowManager.LayoutParams#FLAG_SECURE).
+The core approach `HardwareKeyWatcher` in this lib does not work on all devices.
+There are some failed cases listed as below:
 
```

Comment:
@lydiasama This is my intention, markdown syntax supports it to have short sentences for each line, but after rendering, it's still in 1 paragraph.

![image](https://user-images.githubusercontent.com/16315358/131331090-1c38bbc2-2610-432a-b65e-54af3e59de69.png)

Reply:
I mean to the line:77 😅  @luongvo

Reply:
@lydiasama ah, I think we should keep the habit to add a linebreak between each different markdown section. Don't worry, it will not render a redundant linebreak 🤓